tests/http_integration.rs

```diff
@@ -1,1278 +1,1304 @@
-// cargo test --package surreal --bin surreal --no-default-features --features storage-mem,http --test http_integration -- --nocapture
+// RUST_LOG=warn cargo make ci-http-integration
 mod common;
 
-use std::time::Duration;
```

Comment:
Skip this file during review: only moved the tests into `mod http_integration` so it's easier to scope down the `cargo test` command